src/showplanner/Player.tsx

```diff
@@ -68,6 +69,19 @@ export function Player({ id }: { id: number }) {
   const proMode = useSelector((state: RootState) => state.settings.proMode);
   const dispatch = useDispatch();
 
+  const VUsource =
+    (id: number) => {
+      switch(id) {
+        case 0:
+          return "player-0";
+        case 1:
+          return "player-1";
+        case 2:
+          return "player-2";
+        default:
+          return "master";
```

Comment:
hm, personally I prefer to have my default case throw an error, as it indicates that you may have either forgotten a case or passing an invalid value - so this would use an actual value (`0`?) for the master level and have that as an explicit `case`.

Reply:
Fixed in f3ae3fdb193f3239684cd311bbfdfd5f490c6765